types/sequelize/index.d.ts

```diff
@@ -5752,7 +5751,7 @@ declare namespace sequelize {
         /**
          * A reference to the deferrable collection. Use this to access the different deferrable options.
          */
-        Deferrable: Deferrable;
+        Deferrable:  DeferrableInitiallyDeferred | DeferrableInitiallyImmediate | DeferrableNot | DeferrableSetDeferred | DeferrableSetImmediate;
```

Comment:
The union-type is wrong, please revert this line.
`Deferrable` is an object with all possible Deferrable's in it, as the `Deferrable`-interface is already specified.
see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/75a48de13e2c9fabd7d48532190e926fdc72ae63/types/sequelize/index.d.ts#L2075

Reply:
Then the tests will run through again, I suppose.